### PR TITLE
Added entry for Nimble Storage volume driver

### PR DESCRIPTION
**Pull request by khudgins**
Added link for Nimble's volume driver in legacy_plugins.md

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview ready!

Built with commit a0d82ca0ea6ac1792316eb6254c1290fc94cfae0

https://deploy-preview-632--docker-docs.netlify.com

File: engine/extend/legacy_plugins.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Plugin
...
 [NetApp Plugin](https://github.com/NetApp/netappdvp) (nDVP)                         | A volume plugin that provides direct integration with the Docker ecosystem for the NetApp storage portfolio. The nDVP package supports the provisioning and management of storage resources from the storage platform to Docker hosts, with a robust framework for adding additional platforms in the future.
 [Netshare plugin](https://github.com/ContainX/docker-volume-netshare)                 | A volume plugin that provides volume management for NFS 3/4, AWS EFS and CIFS file systems.
+[Nimble Storage plugin](https://infosight.nimblestorage.com/InfoSight/app#software/Integration+Kits/Linux+Toolkit+%28NLT%29) | A volume plug-in that integrates seamlessly with high-performance Nimble Storage Unified Flash Fabric arrays. The plug-in abstracts all array volume capabilities (such as dedupe, encryption, performance policies, volume placement etc) to the Docker administrator to allow true self-provisioning of secure multi-tenant volumes and clones.
```
Comment (johndmulhausen) [inline engine/extend/legacy_plugins.md]: You can't access this page without logging in.

**Comment by johndmulhausen** [comment]
Need to use a public-facing page and fix your merge conflict.

File: engine/extend/legacy_plugins.md (hunk 1 above)
Comment (mstanleyjones) [inline engine/extend/legacy_plugins.md]: This reads a bit like marketing copy. Can you reduce the description of Nimble Storage and just link to a (non-login-required) page which talks about its capabilities? The other entries here seem a little (though not much) less jargony.

**Comment by khudgins** [comment]
Updated copy to remove marketspeak. Apologies for the delay.

**Comment by mstanleyjones** [comment]
Oops, I didn't realize there is a conflict that needs to be addressed. Can you address that or do you need me to help you out?

**Comment by khudgins** [comment]
I'm honestly not that familiar with our merge/issue workflow, so some pointers would help.

**Comment by mstanleyjones** [comment]
OK, I took these steps:

```bash

# Add your remote
git remote add khudgins https://github.com/khudgins/docker.github.io

# Fetch info about what objects are in your remote
git fetch khudgins

# Checkout the branch you are using for this PR
git checkout add_nimble_driver_link

# Refresh my notion of the upstream
git fetch upstream

# Rebase on upstream/vnext-engine
git rebase upstream/vnext-engine

# See the conflicts
git status

# A file was added since you first did your PR. We need to get it into your PR.

git mergetool

# Choose the modified file, not the deleted one

# Continue the rebase

git rebase --continue

# It is successful, force-push into your branch

git push -f khudgins add_nimble_driver_link
```

Since I force-pushed, if you wanted to do further work on your branch, you'd need to fetch from your fork and hard-reset your local branch to your fork's branch. If I assume you've named your fork `origin`, then you'd do this:

```bash
git checkout add_nimble_driver_link
git fetch origin
git reset --hard origin/add_nimble_driver_link
```

**Comment by mstanleyjones** [comment]
Note that when I first did this, I was caught in a brain void and rebased against `upstream/vnext-engine` instead of `upstream/vnext-master` by accident. I fixed that. Sorry!

**Comment by mstanleyjones** [comment]
@khudgins we still can't merge this until you use a URL that isn't behind a login link. Please update that and let me know. Thanks!

**Comment by johndmulhausen** [comment]
@mstanleyjones @khudgins This is a great contribution but it relies on information that is behind a login. Can we take the link out and ship, feasibly? Can we find an alternate, public-facing reference to link to? So close!

**Comment by johndmulhausen** [comment]
@khudgins @mstanleyjones Unfortunately we have moved the source of truth for /engine/extend/... to the /docker/docker repo, under "docs" 

Open up a PR with these changes over there! Definitely link back to this one so they have context. :)

**Pull request closed without merging**